tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
     "compilerOptions": {
+        "allowJs": true,
```

Comment:
If this important we can try to get rid of this again but I needed it to make the code compile

Reply:
Interesting. Which parts would not compile?

Reply:
Hm, I just tried again and it seems to work also without this now :)